fs/span-manager/span_manager.go

```diff
@@ -45,6 +45,8 @@ const (
 	fetched
 	// A span is in Uncompressed state when it's uncompressed and its uncompressed content is cached.
 	uncompressed
+	// Default number of tries fetching data from remote and verifying the digest.
+	defaultRetries = 3
```

Comment:
this will conflict with the `iota`. the `uncompressed`'s value is 3 based on its order.

suggest to move it out of this iota const block.